.github/workflows/tests.yaml

```diff
@@ -68,10 +69,14 @@ jobs:
         key: ${{ runner.os }}-pip-${{ hashFiles('setup.py') }}
         restore-keys: |
           ${{ runner.os }}-pip-
+    - name: install libgit2 on Python3.10 in macOS
+      if: matrix.pyv == '3.10.0-rc.1' && matrix.os == 'macos-10.15'
+      run: |
+        brew install libgit2
```

Comment:
Python3.10 only running in `macOS` as libgit2-1.1 is not available in Ubuntu (only >= 21.04 which is not supported in GHA).